src/hiero_sdk_python/client/client.py

```
@@ -145,11 +153,15 @@ def _init_mirror_stub(self) -> None:
         for a configurable mirror address, which should use port 443 for HTTPS connections.
         """
         mirror_address = self.network.get_mirror_address()
-        if mirror_address.endswith(':50212') or mirror_address.endswith(':443'):
-            self.mirror_channel = grpc.secure_channel(mirror_address, grpc.ssl_channel_credentials())
+        if mirror_address.endswith(":50212") or mirror_address.endswith(":443"):
```

Ruff (0.14.14)

[warning] Call `endswith` once with a `tuple`

Merge into a single `endswith` call

(PIE810)